site/helm/installing.md

```diff
@@ -0,0 +1,169 @@
+
+## Prerequisites
+
+You will need to have Kubernetes set up. To get up and running fast,
+you might want to use `minikube` or `kubeadm`. Any other Kubernetes
+setup will work as well though.
+
+*Note:* If you are using `minikube`, be sure to start the
+cluster with `--bootstrapper=kubeadm` so you are using RBAC.
+
+Download Helm:
+
+- On MacOS:
+
+  ```sh
+  brew install kubernetes-helm
+  ```
+
+- On Linux:
+  - Download the [latest release](https://github.com/kubernetes/helm/releases/latest), unpack the
+    tarball and put the binary in your `$PATH`.
+
+Now create a service account and a cluster role binding for Tiller:
+
+```sh
+kubectl -n kube-system create sa tiller
+
+kubectl create clusterrolebinding tiller-cluster-rule \
+    --clusterrole=cluster-admin \
+    --serviceaccount=kube-system:tiller
+```
+
+Deploy Tiller in `kube-system` namespace:
+
+```sh
+helm init --skip-refresh --upgrade --service-account tiller
+```
+
+## Install Weave Flux
+
+Add the Flux repository of Weaveworks:
+
+```sh
+helm repo add weaveworks https://weaveworks.github.io/flux
+```
+
+In this next step you install Weave Flux using `helm`. All you need
+to do is specify your fork of
+[flux-helm-test](https://github.com/weaveworks/flux-helm-test).
+
+ 1. Fork [flux-helm-test](https://github.com/weaveworks/flux-helm-test)
+    on Github and
+ 1. install Weave Flux and its Helm Operator by specifying your fork
+    URL.
+
+*Just make sure you replace `YOURUSER` with your GitHub username in
+the command below:*
+
+```sh
+helm install --name flux \
+--set helmOperator.create=true \
+--set git.url=git@github.com:YOURUSER/flux-helm-test \
+--set git.chartsPath=charts \
+--namespace flux \
+weaveworks/flux
+```
+
+Allow some time for all containers to get up and running. If you're
+impatient, run the following command and see the pod creation
+process.
+
+```sh
+watch kubectl get pods --all-namespaces
+```
+
+You will notice that `flux` and `flux-helm-operator` will start
+turning up in the `flux` namespace.
+
+## Giving write access
+
+For the real benefits of GitOps, Flux will need acccess to your
+git repository to update configuration if necessary. To facilitate
+that you will need to add a deploy key to your fork of the
+repository.
+
+This is pretty straight-forward as Flux generates a SSH key and
+logs the public key at startup. Find the SSH public key with:
+
+```sh
+export FLUX_POD=$(kubectl get pods --namespace flux -l "app=flux,release=flux" -o jsonpath="{.items[0].metadata.name}")
+kubectl -n flux logs $FLUX_POD | grep identity.pub | cut -d '"' -f2
+```
+
+In order to sync your cluster state with git you need to copy the
+public key and create a deploy key with write access on your GitHub
+repository.
+
+Open GitHub, navigate to your fork, go to **Setting > Deploy keys**(`https://github.com/YOURUSER/flux-example/settings/keys/new`),
+click on **Add deploy key**, check **Allow write access**, paste
+the Flux public key and click **Add key**.
```

Comment:
You need to give it a name as well; people will figure it out, but nice to mention it and save them a false step.